win32ss/gdi/ntgdi/freetype.c

```diff
@@ -3317,11 +3317,12 @@ IntRequestFontSize(PDC dc, PFONTGDI FontGDI, LONG lfWidth, LONG lfHeight)
         return FT_Request_Size(face, &req);
     }
 
+    /* See also: https://docs.microsoft.com/en-us/typography/opentype/spec/os2#fsselection */
     if (lfHeight > 0)
     {
         /* case (A): lfHeight is positive */
         Sum = pOS2->usWinAscent + pOS2->usWinDescent;
-        if (Sum == 0)
+        if (Sum == 0 || (pOS2->fsSelection & 0x80))
```

Comment:
Can you define a `FM_SEL_USE_TYPO_METRICS` flag with this value of 0x80 in the code?
I based the name on the flag value "name" given in your MSDN link and from the documented fsSelection flag names from https://docs.microsoft.com/en-us/windows/desktop/api/winddi/ns-winddi-_fontdiff .

Reply:
OK.